rat/ops.py

```diff
@@ -647,6 +1076,19 @@ def __iter__(self):
     def code(self):
         return f"jax.scipy.special.logit({self.subexpr.code()})"
 
+    def __post_init__(self):
+        signatures = {
+            (types.RealType,): types.RealType,
+        }
+        self.out_type = types.get_output_type(signatures, (self.subexpr.out_type,))
+
+    def fold(self):
+        self.subexpr = self.subexpr.fold()
+        if isinstance(self.subexpr, RealConstant):
+            return RealConstant(value=jax.scipy.special.logit(self.subexpr.value))
```

Comment:
I wonder if these should be jax operations or since they execute in the compiler they should be numpy. If tests pass with this, let's roll with this but this might turn into a problem in the future (not clear to me when using jax stuff what is being copied to device memory or what)

Reply:
Since the folding is just for scalars, it should be fine to use numpy.